vignettes/estimate_infections.Rmd

```diff
@@ -18,54 +18,103 @@ knitr::opts_chunk$set(
+  I_{0 < t \leq t_\mathrm{seed}} &= I_0 \exp  \left(r t \right)
 \end{align}
 
-### Within the range of observations
+where $I_{t}$ is the number of latent infections on day $t$, $r$ is the estimate of the initial growth rate, and $I_{obs}$ and $r_{obs}$ are estimated from the first week of observed data: $I_{obs}$ as the mean of reported cases in the first 7 days (or the mean of all cases if fewer than 7 days of data are given), divided by the prior mean reporting fraction if less than 1 (see below); and $r_{obs}$ as the point estimate from fitting a linear model to the first 7 days of data (or all data if fewer than 7 days of data are given),
 
-Where $I_{obs}$ and $r_{obs}$ are estimated from the first week of observed data. For the time window of the observed data infections are then modelled by weighting previous infections by the generation time and scaling by the instantaneous reproduction number. These infections are then convolved to cases by of onset date ($O_t$) and cases by date of report ($D_t$) using log-normal delay distributions (though arbitrary delay distributions can be used this is the most common formulation). This model can be defined mathematically as follows,
+\begin{equation}
+log(C_t) \sim rt
+\end{equation}
+where $C_{t}$ is the number of reported cases on day $t$.
 
-\begin{align}
-  \log R_{t} &= \log R_{t-1} + \mathrm{GP}_t \\
-  I_t &= R_t \sum_{\tau = 1}^{T_I} w(\tau | \mu_{w}, \sigma_{w}) I_{t - \tau} \\
-  O_t &= \sum_{\tau = 0}^{T_O} \xi_{O}(\tau | \mu_{\xi_{O}}, \sigma_{\xi_{O}}) I_{t-\tau} \\
-  D_t &= \alpha \sum_{\tau = 0}^{T_D} \xi_{D}(\tau | \mu_{\xi_{D}}, \sigma_{\xi_{D}}) O_{t-\tau} \\
-  C_t &\sim \mathrm{NB}\left(\omega_{(t \mod 7)}D_t, \phi\right)
-\end{align}
+### Infections
+
+For the time window of the observed data and beyond infections are then modelled by weighting previous infections with the generation time and scaling by the instantaneous reproduction number:
+
+\begin{equation}
+  I_t = R_t \sum_{\tau = 1}^{g_\mathrm{max}} g(\tau | \mu_{g}, \sigma_{g}) I_{t - \tau}
+\end{equation}
+
+where $g(\tau|\mu_{g}, \sigma_{g})$ is the distribution of generation times (with discretised gamma or discretised log normal distributions available as options) with mean (or log mean in the case of lognormal distributions) $\mu_g$, standard deviation (or log standard deviation in the case of lognormal distributions) $\sigma_g$ and maximum $g_\mathrm{max}$.
 
-Where $T_I$ is the maximum generation time, $T_O$ is the maximum delay from infection to case onset, $T_D$ is the maximum delay from case onset to report, and $\alpha$ is the proportion of cases that are ultimately reported (again note these are all optional and arbitrary).
+The distribution of generation times $g$ here represents the probability that somebody who became infectious on day 0 and who infects someone else during their course of infection does so on day $\tau > 0$, assuming that infection cannot happen on day 0.
```

Comment:
Might want to specify what this becomes when the generation time is 1 (i.e log growth)